api/private/cerbos/runtime/v1/runtime.proto

```diff
@@ -218,7 +225,7 @@ message IndexBuildErrors {
   repeated DuplicateDef duplicate_defs = 2;
   repeated LoadFailure load_failures = 3;
   repeated MissingImport missing_imports = 4;
-  repeated string missing_scopes = 5;
+  repeated MissingScope missing_scopes = 5;
```

Comment:
This would be a breaking change in other places. Let's mark it as deprecated and stop populating it but we can't drop the field or change its data type.